core/src/test/scala/org/apache/spark/rdd/RDDSuite.scala

```diff
@@ -897,4 +897,23 @@ class RDDSuite extends FunSuite with SharedSparkContext {
       mutableDependencies += dep
     }
   }
+
+  test("Nested RDDs are not supported (SPARK-5063)") {
```

Comment:
It varies from suite-to-suite; most start with lowercase because they start with method names.  If you look at BroadcastSuite, though, most use uppercase style like I've done here.